static/src/javascripts/projects/common/modules/experiments/tests/remind-me-email.js

```diff
@@ -0,0 +1,194 @@
+                comingUp: {
+                    blindDate: {
+                        subtitle: 'Coming up in next week&rsquo;s Blind Date',
+                        title: '<b>&lsquo;She didn’t judge me for talking about giraffes&rsquo; necks&rsquo;</b>',
+                        description: 'Did digital communications consultant Lucy, 23, click with waitress Charlie, 20?',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_blind-date'
+                    },
+                    experience: {
+                        subtitle: 'Coming up in next week&rsquo;s Experience',
+                        title: '<b>I was stranded in the wilderness for nine days</b>',
+                        description: '&lsquo;I had lost so much weight that I looked like walking skeleton. I was ready to give up&rsquo;',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_experience'
+                    }
+                },
+                seriesDescription: {
+                    blindDate: {
+                        subtitle: null,
+                        title: 'What is <b>Blind Date?</b>',
+                        description: 'Every Saturday we fix up two Guardian readers',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_blind-date'
+                    },
+                    experience: {
+                        subtitle: null,
+                        title: 'What is <b>Experience?</b>',
+                        description: 'Every Saturday a Guardian reader shares their unique story',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_experience'
+                    }
+                }
+            }
+        };
+
+        function isBlindDate() {
+            return window.guardian.config.page.series === 'Blind date';
+        }
+
+        function isExperience() {
+            return window.guardian.config.page.series === 'Experience';
+        }
+
+        function insertSignupBox(templateData) {
+            var params = defaults({
+                svg: svg(envelope, ['submit-input__icon'])
+            }, templateData);
+
+            var $signupBox = $.create(template(remindMeEmailSignup, params));
+
+            return fastdom.write(function () {
+                var tags = $('.submeta');
+                $signupBox.insertBefore(tags);
+                mediator.emit('remind-me-email:insert');
+            });
+        }
+
+
+        function getTemplateData(data) {
+            return window.guardian.config.page.series === 'Blind date' ? data.blindDate : data.experience;
+        }
+
+
+        function completer(complete) {
+            mediator.on('remind-me-email:insert', function () {
+                bean.on(qwery('#remind-me-signup')[0], 'click', function (){
```

Comment:
Ha! Yes I know, this syntax drives me nuts (it's even worse when you need to bind handlers to a set of elements) but `bonzo` doesn't have an `.on()` method - one of the many reasons why we should just use jQuery instead. (Or at least modify our custom `$` function so it can bind events)

👍  for the point about the `.js-` selector